docs/07-lifecycle.md

```diff
@@ -2,8 +2,54 @@
 ## Development
 ## Submit to Marketplace
 ## Provision
+### App Bundle
+App bundle contains `manifest.json` file, `static/` folder and one or several lambda function bundles.
```

Comment:
We should prob. start with stating the purpose of the app bundle, what process uses it, etc. before describing what it is composed of.